doc/api/url.md

```diff
@@ -235,6 +235,18 @@ url.resolve('http://example.com/', '/one')    // 'http://example.com/one'
 url.resolve('http://example.com/one', '/two') // 'http://example.com/two'
 ```
 
+Note that `url.resolve()` method depends on the protocol given in the `from` parameter. Some protocols like `https`, `file`, `ftp`, `wss` and `gopher` have special cases.
```

Comment:
That first sentence is fine, just add a line break or two in there so a single line doesn't go past 80 chars :-)